### PR TITLE
[FIX] website_product_filters: search criteria isn't sending when sort filter changes.

### PR DESCRIPTION
**Pull request by gmlp**
**search criteria isn't sending when sort filter changes.It is fixed sending search criteria using js.**
FIXES: https://github.com/Vauxoo/yoytec/issues/788
BUILD: https://github.com/Vauxoo/yoytec/pull/886

### CODE REVIEW

File: website_product_filters/static/src/js/send_extra_params.js [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
-    $('form.js_attributes input, form.js_attributes select, form.js_extra_parameters select',oe_website_sale).on('change', function () {
-        $(this).closest("form").submit();
+    $('form.js_attributes select').on('change', function () {
+        var $form =$(this).closest("form");
+        var search_query = $('.search-query').val();
+        var input = $("<input>")
+                     .attr("type", "hidden")
+                     .attr("name", "search").val(search_query);
+        $form.append($(input));
+        $form.submit();
```
Comment (nhomar) [inline website_product_filters/static/src/js/send_extra_params.js:11]: Por que pruebas un submit "hiden"? entendí bien el selector?

Comment (gmlp) [inline website_product_filters/static/src/js/send_extra_params.js:11]: @nhomar, this add a hidden field with the search criteria into the form.

Comment (nhomar) [inline website_product_filters/static/src/js/send_extra_params.js:11]: ya sorry. +1

**Comment by oscarolar** [comment]
AFAICS There was no need to add extra JS, you can simply add the `form="form_id"` to any other input outside the form that you want to send on the same HTTP request. 
see an example here
http://www.w3schools.com/html/tryit.asp?filename=tryhtml5_input_form

**Pull request merged**